### ISSUE TITLE
Identifying Cannabis abusers

### ISSUE THREAD

**Issue by kuls1** [MIT-LCP/eicu-website#47]
I am interested in selecting Cannabis abusers population. I am surprised to see that there are no Cannabis abusers in this data set, especially when it's coming from ~400 ICU's. I can identify Cocaine users, but for some reason, there are no cannabis users in this data. I have tried searching using both ICD9 and ICD10 codes. Is this for real? Has anyone else observed this?

Here is my code
```
DROP MATERIALIZED VIEW IF EXISTS diagnosis_icd10 CASCADE;

CREATE MATERIALIZED VIEW diagnosis_icd10 as

(SELECT *, split_part(icd9code, ',', 1) AS icd9
     , split_part(icd9code, ', ', 2) AS icd10
FROM   eicu.diagnosis);

SELECT *
FROM diagnosis_icd10
WHERE icd9 = ANY('{'304.30','304.31','304.32','304.33','305.20','305.21','305.22','305.23',}'); 

-- Also tried searching text in diagnosisstring
SELECT *
FROM eicu.diagnosis
where diagnosisstring similar to '%((marijuana)|(cannabis))%';

```

**Comment by jraffa** [comment]
It's unclear what the aim of your study is, but I am unsure if this approach is an appropriate way to identify "abusers".  The cocaine users I can easily identify are those who have overdosed (are these the same ones you have found?).  

None of the codes you have listed for cannabis are for an overdose, so maybe this isn't as surprising.  

Perhaps a clinician will correct me, but the ICU is unlikely to be a place to be for inpatient cannabis treatment (for addiction, overdose, withdrawal etc), whereas for opiates, cocaine and alcohol it can be a place for these.  If you are looking for an estimate of the prevalence of cannabis use, I would suggest using another dataset.  There will be a lot of variation in data collection practices by hospital, and I would believe particularly for things such as this.


**(Also, please follow the instructions at: http://eicu-crd.mit.edu/help/ to post future questions. This repository is intended for managing website content only).**

**Comment by christinium** [comment]
Cocaine can cause things like heart attacks, hypertensive emergencies,
strokes, muscle death etc and the use of cocaine be the direct cause of an
admission to the ICU.  Even if someone documents recreational use of
Cannabis, I typically wouldn't bill for it unless there is a reason. Have
you searched through the social history portion of the discharge notes?
That is the most likely place where it would be documented. Also, keep in
mind that if the patient comes in half conscious/unconscious because s/he
is very ill, the physician writing the note may not be able get an
excellent social history.  Good luck!

**Comment by kuls1** [comment]
No, I am not planning a prevalence study.  At the moment I want to identify patients with substance use disorder. I am using the ICD9 codes found here: [](https://www.ncbi.nlm.nih.gov/books/NBK310986/table/sb191.t4/)

I was surprised since I was able to identify patients were having ICD9 codes related to Cannabis use in the MIMIC-III data set but not in eICU.  I was able to find patients with ICD9 code 968.5 corresponding to 'Poisoning by cocaine.'

@christinium Thanks for the suggestion to search in the social history portion. Also, I agree with you those patients won't be billed even if Cannabis use was documented. 
Apologies for posting my query in the wrong section.

**Comment by jraffa** [comment]
FYI, No discharge summaries in eICU.

**Comment by kuls1** [comment]
No, I am not planning a prevalence study.  At the moment I want to identify patients with substance use disorder. I am using the ICD9 codes found here: [](https://www.ncbi.nlm.nih.gov/books/NBK310986/table/sb191.t4/)

I was surprised since I was able to identify patients were having ICD9 codes related to Cannabis use in the MIMIC-III data set but not in eICU.  I was able to find patients with ICD9 code 968.5 corresponding to 'Poisoning by cocaine.'

@christinium Thanks for the suggestion to search in the social history portion. Also, I agree with you those patients won't be billed even if Cannabis use was documented. 

Apologies for posting my query in the wrong section.

**Comment by kuls1** [comment]
No, I am not planning a prevalence study.  At the moment I want to identify patients with substance use disorder. I am using the ICD9 codes found here: https://www.ncbi.nlm.nih.gov/books/NBK310986/table/sb191.t4/

I was surprised since I was able to identify patients were having ICD9 codes related to Cannabis use in the MIMIC-III data set but not in eICU.  I was able to find patients with ICD9 code 968.5 corresponding to 'Poisoning by cocaine.'

@christinium Thanks for the suggestion to search in the social history portion. Also, I agree with you those patients won't be billed even if Cannabis use was documented. 

Apologies for posting my query in the wrong section.

**Comment by christinium** [comment]
Sorry! Didn't read the subject carefully! Jesse is right, there are no
discharge notes for the eICU unfortunately!
Christina

**Comment by jraffa** [comment]
Poisoning ~ overdose. 

Again, I don't know what you plan to do in your study, but I wouldn't assume you are pulling out people with a 'substance abuse disorder', but rather those who had a documented overdose, or documented inpatient withdrawal.
